Description: adapt classpath and desktop icon
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2024-03-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/MANIFEST.MF
+++ b/MANIFEST.MF
@@ -1,4 +1,4 @@
 Manifest-Version: 1.0
 Main-Class: net.sourceforge.dvb.projectx.common.Start
 Created-By: 1.2.2 (Sun Microsystems Inc.)
-Class-Path: lib/commons-net-1.3.0.jar libjakarta-oro-2.0.8.jar
+Class-Path: /usr/share/java/commons-net.jar /usr/share/java/oro.jar
--- a/projectx.desktop
+++ b/projectx.desktop
@@ -1,8 +1,8 @@
 [Desktop Entry]
 X-SuSE-translate=true
-Categories=AudioVideo;AudioVideoEditing;
+Categories=AudioVideoEditing;
 Comment=A video editing and demultiplexing tool
-Exec=java -jar /usr/share/projectx/ProjectX.jar
+Exec=/usr/bin/projectx
 GenericName=ProjectX
 Icon=package_multimedia
 Name=ProjectX
